main.css

```
@@ -1,25 +1,83 @@
-A:link, A:visited, A:active, A:hover {
-  text-decoration: none; font-weight: bold;
-}
-A:HOVER { color: "c0c0ff"; }
-#SUBTXT { font-size: 10pt; font-family: myriad,meta,arial,helvetica; }
-#MIC { font-size: 4pt; font-family: myriad,meta,arial,helvetica; }
-#H1 { font-size: 12pt; font-family: myriad,meta,arial,helvetica;
-      font-weight: bold; }
-#H2 { font-size: 14pt; font-family: myriad,meta,arial,helvetica;
-      font-weight: bold; }
-
-BODY {
+@media screen and (min-width: 50rem) {
+  body {
+    margin: 0.25rem auto 1rem auto;
+    padding: 1rem;
+    max-width: 70rem;
+  }
+}
+
+body, div, table, tt, td, thead, tbody, tr, span, a {
   font-size: 11pt; 
-  font-family: myriad,meta,arial,helvetica; 
+  font-family: -apple-system, BlinkMacSystemFont, “Segoe UI”, Roboto, Helvetica, Arial, sans-serif;
+}
+
+body {
   aSCROLLBAR-TRACK-COLOR: #3188ff;
   aSCROLLBAR-3DLIGHT-COLOR: #d2f2ff;
   SCROLLBAR-HIGHLIGHT-COLOR: #000000;
   SCROLLBAR-ARROW-COLOR: #d2f2ff;
   SCROLLBAR-DARKSHADOW-COLOR: #3188ff;
   SCROLLBAR-BASE-COLOR: #000000;
- }  
-ALL, P, TD { font-size: 11pt; 
-  font-family: myriad,meta,arial,helvetica; }
-TH { font-size: 12pt; font-family: myriad,meta,arial,helvetica;
-  font-weight: bold; }
+}
+
+a:link, 
+a:visited, 
+a:active, 
+a:hover {
+  text-decoration: none; 
+  font-weight: bold;
+}
+
+a:hover { 
+  color: "c0c0ff"; 
+}
+
+#SUBTXT { 
+  font-size: 10pt; 
+}
+
+#MIC { 
+  font-size: 4pt; 
+}
+
+#H1 { 
+  font-size: 12pt;
+  font-weight: bold; 
+}
+
+#H2 { 
+  font-size: 14pt; 
+  font-weight: bold; 
+}
+
+ALL, p, td { 
+  font-size: 11pt; 
+}
+
+th { 
+  font-size: 12pt; 
+  font-weight: bold; 
+}
+
+pre,
+code {
+  font-family: SFMono-Regular, Menlo, Monaco, "Courier New", monospace;
+}
+
+tt {
+  color: #000080;
+  font-family: SFMono-Regular, Menlo, Monaco, "Courier New", monospace;
+} 
+
+#RC {
+  background-color: #e0e0ff; 
+  border: 2px solid #c0c0ff; 
+  padding: 10px;
+  margin: 7px;
+}
+
+.important {
+    background: #ffe0e0; 
+    padding: 1rem;
+    border-radius: 5px;
+}
```

Add an empty line at the end of the file
```suggestion
}

```